diff -Nur freeswitch-1.4.13.orig/libs/freetdm/src/ftdm_backtrace.c freeswitch-1.4.13/libs/freetdm/src/ftdm_backtrace.c
--- freeswitch-1.4.13.orig/libs/freetdm/src/ftdm_backtrace.c	2014-12-12 01:25:26.688588038 +0800
+++ freeswitch-1.4.13/libs/freetdm/src/ftdm_backtrace.c	2014-12-12 03:29:44.463307939 +0800
@@ -2,6 +2,9 @@
  *
  *
  */
+#ifndef _DEFAULT_SOURCE
+#define _DEFAULT_SOURCE
+#endif
 #define _BSD_SOURCE
 #include "private/ftdm_core.h"
 
diff -Nur freeswitch-1.4.13.orig/src/include/switch.h freeswitch-1.4.13/src/include/switch.h
--- freeswitch-1.4.13.orig/src/include/switch.h	2014-12-12 01:25:27.321588014 +0800
+++ freeswitch-1.4.13/src/include/switch.h	2014-12-12 04:21:24.024191526 +0800
@@ -60,8 +60,8 @@
 #endif
 #endif
 #ifdef __linux__
-#ifndef _BSD_SOURCE
-#define _BSD_SOURCE
+#ifndef _DEFAULT_SOURCE
+#define _DEFAULT_SOURCE
 #endif
 #endif
 #ifndef __BSD_VISIBLE
