diff -Nur freeswitch-1.4.13.orig/libs/freetdm/src/ftdm_backtrace.c freeswitch-1.4.13/libs/freetdm/src/ftdm_backtrace.c
--- freeswitch-1.4.13.orig/libs/freetdm/src/ftdm_backtrace.c	2014-12-12 01:25:26.688588038 +0800
+++ freeswitch-1.4.13/libs/freetdm/src/ftdm_backtrace.c	2014-12-12 01:27:49.549582672 +0800
@@ -2,6 +2,9 @@
  *
  *
  */
+#ifndef _DEFAULT_SOURCE
+#define _DEFAULT_SOURCE
+#endif
 #define _BSD_SOURCE
 #include "private/ftdm_core.h"
 
