--- uwsgiconfig.py	2011-09-11 13:06:59.000000000 +0800
+++ uwsgiconfig.new.py	2011-10-15 23:35:42.138703475 +0800
@@ -219,7 +219,7 @@
         if uwsgi_os == 'Linux':
             self.gcc_list.append('lib/linux_ns')
             self.gcc_list.append('lib/netlink')
-        self.cflags = ['-O2', '-Wall', '-Werror', '-D_LARGEFILE_SOURCE', '-D_FILE_OFFSET_BITS=64'] + os.environ.get("CFLAGS", "").split()
+        self.cflags = ['-O2', '-Wall', '-D_LARGEFILE_SOURCE', '-D_FILE_OFFSET_BITS=64'] + os.environ.get("CFLAGS", "").split()
 
         try:
             gcc_version = str(spcall("%s -dumpversion" % GCC))
