diff -urN pjproject-2.1.0-orig/pjmedia/src/pjmedia/resample_libsamplerate.c pjproject-2.1.0/pjmedia/src/pjmedia/resample_libsamplerate.c
--- pjproject-2.1.0-orig/pjmedia/src/pjmedia/resample_libsamplerate.c	2013-03-25 21:00:56.718643953 -0400
+++ pjproject-2.1.0/pjmedia/src/pjmedia/resample_libsamplerate.c	2013-03-25 21:01:29.310647197 -0400
@@ -33,7 +33,7 @@
 
 #if PJMEDIA_RESAMPLE_IMP==PJMEDIA_RESAMPLE_LIBSAMPLERATE
 
-#include "../../third_party/libsamplerate/src/samplerate.h"
+#include "samplerate.h"
 
 #define THIS_FILE   "resample_libsamplerate.c"
 
